juju/model.py

```diff
@@ -1072,8 +1072,6 @@ def _get_series(self, entity_url, entity):
                     application_name = entity['Meta']['charm-metadata']['Name']
                 if not series:
                     series = self._get_series(entity_url, entity)
-                if not channel:
-                    channel = 'stable'
```

Comment:
Okay, makes sense, I just wanted to make sure the subsequent call to AddCharm wouldn't fail when passed a None channel.